### ISSUE TITLE
Add flags for testing against different levels of WCAG

### ISSUE THREAD

**Issue by joshiefishbein** [sanctuarycomputer/candelabra#12]
I should be able to do something of the effect `--level=aa` to test my site for violations within the Level AA scope.

Do for Level A, AA, and AAA.
